### PR TITLE
[PAL N64] Match actor changes that aren't in NTSC 1.2 or GC releases

### PR DESCRIPTION
**Pull request by cadmic**
These changes are unrelated to 50 Hz (I think?) so I'm not sure why they didn't make it to NTSC 1.2 or GC releases

### CODE REVIEW

File: src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c [hunk 1]
```diff
@@ -91,10 +92,17 @@ void BgSpot07Taki_Draw(Actor* thisx, PlayState* play) {
...
             gSPSegment(POLY_XLU_DISP++, 0x09,
                        Gfx_TwoTexScroll(play->state.gfxCtx, G_TX_RENDERTILE, ((frames * -1) & 0x7F),
                                         ((frames * -3) & 0xFF), 64, 64, 1, ((frames * 1) & 0x7F),
                                         ((frames * -3) & 0xFF), 64, 64));
+#else
+            gSPSegment(POLY_XLU_DISP++, 0x09,
+                       Gfx_TwoTexScroll(play->state.gfxCtx, G_TX_RENDERTILE, ((frames * -1) & 0xFF),
+                                        ((frames * -3) & 0xFF), 64, 64, 1, ((frames * 1) & 0xFF),
+                                        ((frames * -3) & 0xFF), 64, 64));
+#endif
```
Comment (fig02) [inline src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: I would think this is framerate related, its modifying the way the waterfall texture is scrolling. My guess is that the mod value used in ntsc result in unnatural snapping at 60hz and they had to make the scrolling last longer before resetting the cycle on 50hz. but did not confirm anything

Comment (cadmic) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: wouldn't `frames` here be synced to the frame rate at both 60hz and 50hz?

Comment (fig02) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: oh I guess it would be. not sure then

Comment (krm01) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: this is kinda baffling. usually 64x64 textures are always scrolled `% 256`, aka `& 0xFF`. this is the only instance in the codebase where it's being scrolled `% 127` / `& 0x7F`

Comment (cadmic) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: Maybe they noticed that and fixed the bug, but only for PAL for some reason? tbh I'm not sure how these changes didn't make it to 1.2 given that most of the PAL 1.0 bugfixes did. Maybe whoever was working on this didn't get the memo that there would be another NTSC release and wrapped it in `#if PAL`?

Comment (mdtauk) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: IIRC PAL releases were delayed for localisation so any bugs found after NTSC and JP went Gold, could only be fixed for later ROM editions, or PAL releases.

Comment (fig02) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: 1.2 came after PAL 1.0, we know this based on build date. Theres no reason a fix like this should be in PAL 1.0 and not NTSC 1.2.

Comment (cadmic) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: The counterpart to this is #2201, which are changes in NTSC 1.2 that are not in PAL 1.0. In particular the ganon battle start radius change seems to be a fix from PAL testing (https://discord.com/channels/688807550715560050/688851317593997489/1284251020150444093). But, it's possible the releases were built from different branches or something so history does not go strictly in order

Comment (cadmic) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c:105]: Sorry to keep blabbing about this, but it's also weird that the changes in this PR did not make it into GC releases, but the changes for NTSC 1.2 did make it into GC releases. Maybe that's evidence that PAL 1.0/1.1 was built from a slightly different branch, and these fixes were only in that branch

Comment (fig02) [reply src/overlays/actors/ovl_Bg_Spot07_Taki/z_bg_spot07_taki.c]: no need to apologize, this is interesting to think about :) 
I think i would agree

**Review by fig02 — APPROVED** [review]

**Review by krm01 — APPROVED** [review]

**Pull request merged**